python/lsst/ts/m2com/mock/mock_control_closed_loop.py

```diff
@@ -641,7 +643,8 @@ def _calc_look_up_forces_temperature(self, lut_temperature, temperature_ref):
             Temperature compensation vector for uniform gradient.
         """
 
-        temperature = lut_temperature - temperature_ref
+        indexs = [11, 0, 1, 2, 3, 6, 7, 8, 5, 4, 10, 9]
+        temperature = lut_temperature - temperature_ref[indexs]
```

Comment:
Shouldn't you better keep the same order for temperature_ref and lut_temperature? So you don't have to reindex temperature_red? Also, keep naming consistent - e.g. either lut_temperature and ref_temperature or temperature_lut and temperature_ref (I prefer the first one).

Reply:
This is because Bo defined the `ts_xml` to have the order of  `temperature_ref` but the LUT calculation by vendor has the another order. The order in `ts_xml` is more straightforward to the user when using the CSC to change the temperature offsets. A possible solution is to modify the LUT in the calculation to have the same order as defined in `ts_xml`. But this will take too much effort at this moment. Thanks!

Reply:
Renamed the variables in commit: [b24e0d5](https://github.com/lsst-ts/ts_m2com/pull/15/commits/b24e0d5d230d46d96dc3e647896c54f8579e82da). Thanks!